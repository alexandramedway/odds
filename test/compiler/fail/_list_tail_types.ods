do t = tail([1, 2, 3])
do head(t) || true
