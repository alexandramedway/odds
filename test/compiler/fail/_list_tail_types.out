[31mSemantic error[0m:
  Invalid use of binary operator '||' with types Num and Bool
